docs/source/gbd2019_models/causes/pad/cause_hierarchy_pad.svg

```diff
@@ -0,0 +1,3 @@
 xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 331px; margin-left: 292px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">Other</div></div></div></foreignObject><text x="351" y="335" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">Other</text></switch></g><rect x="1" y="402" width="120" height="120" fill="none" stroke="#000000" stroke-width="3" stroke-dasharray="9 9" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 462px; margin-left: 2px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><div>1 sequelae: symptomatic peripheral arterial disease</div><div>s_946<br /></div></div></div></div></foreignObject><text x="61" y="466" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">1 sequelae: symptoma...</text></switch></g><rect x="171" y="402" width="120" height="120" fill="none" stroke="#000000" stroke-width="3" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 462px; margin-left: 172px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><div>1 sequela:</div><div>asymptomatic peripheral arterial disease</div><div>s_1041</div></div></div></div></foreignObject><text x="231" y="466" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">1 sequela:...</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
I think the sequela_id in this diagram should actually be 964, not 946

Reply:
Oh thank you! I will change that!